packages/create-instance/create-component-stubs.js

```diff
@@ -5,8 +5,8 @@ import {
   throwError,
   camelize,
   capitalize,
-  hyphenate,
-  keys
+  hyphenate
+  // keys
```

Comment:
Commented out for formatting until we can figure out how to merge everything in.